### ISSUE TITLE
mapfishapp - update the CSW querier in order to make it compliant to CSW spec and geocatalogue.fr types

### ISSUE THREAD

**Issue by bchartier** [georchestra/georchestra#1785]
The CSW queries sent to french geocatalogue.fr never retrieves records (see https://groups.google.com/forum/#!topic/georchestra/eUe_--owc20).
This seems to be due to the filter on the property "Type":
```
<ogc:Or>
    <ogc:PropertyIsLike wildCard="*" singleChar="." escapeChar="!">
        <ogc:PropertyName>Type</ogc:PropertyName>
        <ogc:Literal>dataset</ogc:Literal>
    </ogc:PropertyIsLike>
    <ogc:PropertyIsLike wildCard="*" singleChar="." escapeChar="!">
        <ogc:PropertyName>Type</ogc:PropertyName>
        <ogc:Literal>series</ogc:Literal>
    </ogc:PropertyIsLike>
</ogc:Or>
```

CSW specifications defines the following values: "dataset" and "datasetcollection".
Whereas geocatalogue.fr only seems to understand "data".
"dataset" and "series" seem to work well with geonetwork catalogues.

Therefore, I suggest to use the following filter:
```
<ogc:Or>
    <ogc:PropertyIsEqualTo>
        <ogc:PropertyName>Type</ogc:PropertyName>
        <ogc:Literal>data</ogc:Literal>
    </ogc:PropertyIsEqualTo>
    <ogc:PropertyIsEqualTo>
        <ogc:PropertyName>Type</ogc:PropertyName>
        <ogc:Literal>dataset</ogc:Literal>
    </ogc:PropertyIsEqualTo>
    <ogc:PropertyIsEqualTo>
        <ogc:PropertyName>Type</ogc:PropertyName>
        <ogc:Literal>datasetcollection</ogc:Literal>
    </ogc:PropertyIsEqualTo>
    <ogc:PropertyIsEqualTo>
        <ogc:PropertyName>Type</ogc:PropertyName>
        <ogc:Literal>series</ogc:Literal>
    </ogc:PropertyIsEqualTo>
</ogc:Or> 
```

In order to do this I suggest to replace the lines https://github.com/georchestra/georchestra/blob/16.12/mapfishapp/src/main/webapp/app/js/GEOR_cswquerier.js#L838-L849 by:
```
                        filters: [
                            new OpenLayers.Filter.Comparison({
                                type: "==",
                                property: "Type",
                                value: 'data'
                            }),
                            new OpenLayers.Filter.Comparison({
                                type: "==",
                                property: "Type",
                                value: 'dataset'
                              }),
                             new OpenLayers.Filter.Comparison({
                                type: "==",
                                property: "Type",
                                value: 'datasetcollection'
                              }),
                             new OpenLayers.Filter.Comparison({
                                type: "==",
                                property: "Type",
                                value: 'series'
                            })
                        ]
```

Off course, this has not been tested (tested in CSW queries, but not in mapfishapp source code).
Sorry for not providing a PR.

**Comment by fvanderbiest** [comment]
Thanks for your investigation Benjamin.
A fix was pushed to 15.12 with 3a78554cd and also ported to later branches.

**Comment by bchartier** [comment]
Thanks a lot.
